components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -572,15 +572,14 @@ struct JupiterErrorResponse {
 const string kLedgerHardwareVendor = "Ledger";
 const string kTrezorHardwareVendor = "Trezor";
 
-const string kFilecoinKeyringId = "filecoin";
-const string kFilecoinTestnetKeyringId = "filecoin_testnet";
-
-const string kSolanaKeyringId = "solana";
-const string kDefaultKeyringId = "default";
-
-// BIP 84 keyrings.
-const string kBitcoinKeyring84Id = "bitcoin_84";
-const string kBitcoinKeyring84TestId = "bitcoin_84_test";
+enum KeyringId {
+  kDefaultKeyringId = 0,
+  kSolanaKeyringId = 1,
+  kFilecoinKeyringId = 2,
+  kFilecoinTestnetKeyringId = 3,
+  kBitcoinKeyring84Id = 4,
+  kBitcoinKeyring84TestId = 5,
+};
```

Comment:
Let's take this opportunity to come up with more concise naming (see an [example](https://google.github.io/styleguide/cppguide.html#Enumerator_Names) from the Google C++ style guide).

```suggestion
enum Keyring {
  kDefault = 0,
  kSolana = 1,
  kFilecoin = 2,
  kFilecoinTestnet = 3,
  kBitcoin84 = 4,
  kBitcoin84Test = 5,
};
```